contracts/governance/extensions/GovernorOverrideDelegateVote.sol

```diff
@@ -0,0 +1,181 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {GovernorVotes} from "./GovernorVotes.sol";
+import {IVotes} from "../utils/IVotes.sol";
+import {IERC5805} from "../../interfaces/IERC5805.sol";
+import {SafeCast} from "../../utils/math/SafeCast.sol";
+import {Time} from "../../utils/types/Time.sol";
+import {VotesOverridable} from "../utils/VotesOverridable.sol";
+
+/**
+ * @dev Extension of {Governor} which enables delegatees to override the vote of their delegates. This module requires a
+ * token token that inherits `VotesOverridable`.
+ */
+abstract contract GovernorOverrideDelegateVote is GovernorVotes {
```

Comment:
If we agree this is a voting module, then we should probably name it in a way that is succint yet explicit about what it achieves. I think the current name may be a bit too long.

This is just a proposition:
```suggestion
abstract contract GovernorCountOverridable is GovernorVotes {
```